ui/src/app/applications/components/applications-list/applications-table.tsx

```diff
@@ -118,22 +119,32 @@ export const ApplicationsTable = (props: {
                                                     </div>
                                                 </div>
                                             </div>
+
                                             <div className='columns small-2'>
-                                                <AppUtils.HealthStatusIcon state={app.status.health} /> <span>{app.status.health.status}</span> <br />
-                                                <AppUtils.ComparisonStatusIcon status={app.status.sync.status} />
-                                                <span>{app.status.sync.status}</span> <OperationState app={app} quiet={true} />
-                                                <DropDownMenu
-                                                    anchor={() => (
-                                                        <button className='argo-button argo-button--light argo-button--lg argo-button--short'>
-                                                            <i className='fa fa-ellipsis-v' />
-                                                        </button>
-                                                    )}
-                                                    items={[
-                                                        {title: 'Sync', action: () => props.syncApplication(app.metadata.name, app.metadata.namespace)},
-                                                        {title: 'Refresh', action: () => props.refreshApplication(app.metadata.name, app.metadata.namespace)},
-                                                        {title: 'Delete', action: () => props.deleteApplication(app.metadata.name, app.metadata.namespace)}
-                                                    ]}
-                                                />
+                                                <div className='row'>
```

Comment:
I feel the table has too many columns to display and it's slightly cramped, I would prefer to have created time in the tooltip
![Screen Shot 2022-11-30 at 12 23 54 PM](https://user-images.githubusercontent.com/11219262/204900639-0714db6a-bfbe-4d7d-80d5-f05ebe2ab2f4.png)

Reply:
I agree with you, but where exactly in the tooltip? or you mean as a tooltip of one table row?

Reply:
I guess the tooltip on the table row with additional metadata would be better however if you just want to display `age` then I would prefer tooltip on the application name